modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -34,36 +34,75 @@ Establishing the right performance expectations refers to the fact that the real
 == Provisioning a worker with real-time capabilities
 
 . Install Performance Addon Operator to the cluster.
+
 . Optional: Add a node to the {product-title} cluster.
 See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/html-single/tuning_guide/index#Setting_BIOS_parameters[Setting BIOS parameters].
-. Optional: Create a new machine config pool for real-time nodes.
-. Add the node to the proper machine config pool, using node role labels.
-+
-You must decide which nodes will be configured with real-time workloads. It could be all of the nodes in the cluster or a subset of the nodes. The Performance Addon Operator expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
 
-. Create the `PerformanceProfile` with the proper set of housekeeping cores and `realTimeKernel: enabled: true`.
+. Add the label `worker-rt` to the worker nodes that require the real-time capability by using the `oc` command.
 
-. Specify a node selector in the `PerformanceProfile`, as shown here:
+. Create a new machine config pool for real-time nodes:
 +
 [source,yaml]
 ----
-apiVersion: performance.openshift.io/v2
-kind: PerformanceProfile
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
 metadata:
- name: example-performanceprofile
+  name: worker-rt
+  labels:
+    machineconfiguration.openshift.io/role: worker-rt
 spec:
-...
-  realTimeKernel:
-    enabled: true
+  machineConfigSelector:
+    matchExpressions:
+      - {
+           key: machineconfiguration.openshift.io/role,
+           operator: In,
+           values: [worker, worker-rt],
+        }
+  paused: false
   nodeSelector:
-    node-role.kubernetes.io/worker-rt: ""
+    matchLabels:
+      node-role.kubernetes.io/worker-rt: ""
 ----
+Note that a machine config pool worker-rt is created for group of nodes that have the label `worker-rt`.
+
+. Add the node to the proper machine config pool by using node role labels.
++
+[NOTE]
+====
+You must decide which nodes are configured with real-time workloads. You could configure all of the nodes in the cluster, or a subset of the nodes. The Performance Addon Operator that expects all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, you just point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
```

Comment:
```suggestion
You must decide which nodes are configured with real-time workloads. You could configure all of the nodes in the cluster, or a subset of the nodes. 

The Performance Addon Operator expects that all of the nodes are part of a dedicated machine config pool. If you use all of the nodes, point the Performance Addon Operator to the worker node role label. If you use a subset, you must group the nodes into a new machine config pool.
```

Not sure if there are better words/phrases for "expects" and "points."

Reply:
expects >> I see that we use it in our docs e.g. The Operator expects this mapping file in a predefined location a.....
points>> we use that as well e.g   you can point to a custom tailoring file. >> here I can phrase it as `you can point the Performance.....`

Reply:
no wait , its not optional , you must is more apt here